apps/vr-tests-react-components/package.json

```diff
@@ -56,8 +56,9 @@
     "@fluentui/react-storybook-addon": "*",
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28582&commitId=70d346c027c5e9cb750c3e0f92905f6dde264853&env=prod&clientType=FLUENTUI)


  <details><summary><b>InteractionTag Converged</b> 73 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InteractionTag Converged.Brand - Dark Mode.default.chromium.png|0| Added|
|InteractionTag Converged.Brand - Dark Mode.hover content.chromium.png|0| Added|
|InteractionTag Converged.Brand - Dark Mode.hover dismiss.chromium.png|0| Added|
|InteractionTag Converged.Brand - Dark Mode.pressed content.chromium.png|0| Added|
|InteractionTag Converged.Brand - Dark Mode.pressed dismiss.chromium.png|0| Added|
|InteractionTag Converged.Brand - High Contrast.default.chromium.png|0| Added|
|InteractionTag Converged.Brand - High Contrast.hover content.chromium.png|0| Added|
|InteractionTag Converged.Brand - High Contrast.hover dismiss.chromium.png|0| Added|
|InteractionTag Converged.Brand - High Contrast.pressed content.chromium.png|0| Added|
|InteractionTag Converged.Brand - High Contrast.pressed dismiss.chromium.png|0| Added|
|InteractionTag Converged.Brand.default.chromium.png|0| Added|
|InteractionTag Converged.Brand.hover content.chromium.png|0| Added|
|InteractionTag Converged.Brand.hover dismiss.chromium.png|0| Added|
|InteractionTag Converged.Brand.pressed content.chromium.png|0| Added|
|InteractionTag Converged.Brand.pressed dismiss.chromium.png|0| Added|
|InteractionTag Converged.Circular Dismissible - RTL.default.chromium.png|0| Added|
|InteractionTag Converged.Circular Dismissible - RTL.focus content.chromium.png|0| Added|
|InteractionTag Converged.Circular Dismissible - RTL.focus dismiss.chromium.png|0| Added|
|InteractionTag Converged.Circular Dismissible.default.chromium.png|0| Added|
|InteractionTag Converged.Circular Dismissible.focus content.chromium.png|0| Added|
|InteractionTag Converged.Circular Dismissible.focus dismiss.chromium.png|0| Added|
|InteractionTag Converged.Circular With Media - RTL.default.chromium.png|0| Added|
|InteractionTag Converged.Circular With Media - RTL.focus content.chromium.png|0| Added|
|InteractionTag Converged.Circular With Media.default.chromium.png|0| Added|
|InteractionTag Converged.Circular With Media.focus content.chromium.png|0| Added|
|InteractionTag Converged.Circular With Secondary Text.default.chromium.png|0| Added|
|InteractionTag Converged.Circular With Secondary Text.focus content.chromium.png|0| Added|
|InteractionTag Converged.Circular.default.chromium.png|0| Added|
|InteractionTag Converged.Circular.focus content.chromium.png|0| Added|
|InteractionTag Converged.Filled - Dark Mode.default.chromium.png|0| Added|
|InteractionTag Converged.Filled - Dark Mode.hover content.chromium.png|0| Added|
|InteractionTag Converged.Filled - Dark Mode.hover dismiss.chromium.png|0| Added|
|InteractionTag Converged.Filled - Dark Mode.pressed content.chromium.png|0| Added|
|InteractionTag Converged.Filled - Dark Mode.pressed dismiss.chromium.png|0| Added|
|InteractionTag Converged.Filled - High Contrast.default.chromium.png|0| Added|
|InteractionTag Converged.Filled - High Contrast.hover content.chromium.png|0| Added|
|InteractionTag Converged.Filled - High Contrast.hover dismiss.chromium.png|0| Added|
|InteractionTag Converged.Filled - High Contrast.pressed content.chromium.png|0| Added|
|InteractionTag Converged.Filled - High Contrast.pressed dismiss.chromium.png|0| Added|
|InteractionTag Converged.Filled.default.chromium.png|0| Added|
|InteractionTag Converged.Filled.hover content.chromium.png|0| Added|
|InteractionTag Converged.Filled.hover dismiss.chromium.png|0| Added|
|InteractionTag Converged.Filled.pressed content.chromium.png|0| Added|
|InteractionTag Converged.Filled.pressed dismiss.chromium.png|0| Added|
|InteractionTag Converged.Outline - Dark Mode.default.chromium.png|0| Added|
|InteractionTag Converged.Outline - Dark Mode.hover content.chromium.png|0| Added|
|InteractionTag Converged.Outline - Dark Mode.hover dismiss.chromium.png|0| Added|
|InteractionTag Converged.Outline - Dark Mode.pressed content.chromium.png|0| Added|
|InteractionTag Converged.Outline - Dark Mode.pressed dismiss.chromium.png|0| Added|
|InteractionTag Converged.Outline - High Contrast.default.chromium.png|0| Added|
|InteractionTag Converged.Outline - High Contrast.hover content.chromium.png|0| Added|
|InteractionTag Converged.Outline - High Contrast.hover dismiss.chromium.png|0| Added|
|InteractionTag Converged.Outline - High Contrast.pressed content.chromium.png|0| Added|
|InteractionTag Converged.Outline - High Contrast.pressed dismiss.chromium.png|0| Added|
|InteractionTag Converged.Outline.default.chromium.png|0| Added|
|InteractionTag Converged.Outline.hover content.chromium.png|0| Added|
|InteractionTag Converged.Outline.hover dismiss.chromium.png|0| Added|
|InteractionTag Converged.Outline.pressed content.chromium.png|0| Added|
|InteractionTag Converged.Outline.pressed dismiss.chromium.png|0| Added|
|InteractionTag Converged.Rounded Dismissible - RTL.default.chromium.png|0| Added|
|InteractionTag Converged.Rounded Dismissible - RTL.focus content.chromium.png|0| Added|
|InteractionTag Converged.Rounded Dismissible - RTL.focus dismiss.chromium.png|0| Added|
|InteractionTag Converged.Rounded Dismissible.default.chromium.png|0| Added|
|InteractionTag Converged.Rounded Dismissible.focus content.chromium.png|0| Added|
|InteractionTag Converged.Rounded Dismissible.focus dismiss.chromium.png|0| Added|
|InteractionTag Converged.Rounded With Media - RTL.default.chromium.png|0| Added|
|InteractionTag Converged.Rounded With Media - RTL.focus content.chromium.png|0| Added|
|InteractionTag Converged.Rounded With Media.default.chromium.png|0| Added|
|InteractionTag Converged.Rounded With Media.focus content.chromium.png|0| Added|
|InteractionTag Converged.Rounded With Secondary Text.default.chromium.png|0| Added|
|InteractionTag Converged.Rounded With Secondary Text.focus content.chromium.png|0| Added|
|InteractionTag Converged.Rounded.default.chromium.png|0| Added|
|InteractionTag Converged.Rounded.focus content.chromium.png|0| Added|

</details>


  <details><summary><b>Tag Converged</b> 21 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tag Converged.Brand - Dark Mode.default.chromium.png|0| Added|
|Tag Converged.Brand - High Contrast.default.chromium.png|0| Added|
|Tag Converged.Brand.default.chromium.png|0| Added|
|Tag Converged.Circular Dismissible - RTL.default.chromium.png|0| Added|
|Tag Converged.Circular Dismissible.default.chromium.png|0| Added|
|Tag Converged.Circular With Media - RTL.default.chromium.png|0| Added|
|Tag Converged.Circular With Media.default.chromium.png|0| Added|
|Tag Converged.Circular With Secondary Text.default.chromium.png|0| Added|
|Tag Converged.Circular.default.chromium.png|0| Added|
|Tag Converged.Filled - Dark Mode.default.chromium.png|0| Added|
|Tag Converged.Filled - High Contrast.default.chromium.png|0| Added|
|Tag Converged.Filled.default.chromium.png|0| Added|
|Tag Converged.Outline - Dark Mode.default.chromium.png|0| Added|
|Tag Converged.Outline - High Contrast.default.chromium.png|0| Added|
|Tag Converged.Outline.default.chromium.png|0| Added|
|Tag Converged.Rounded Dismissible - RTL.default.chromium.png|0| Added|
|Tag Converged.Rounded Dismissible.default.chromium.png|0| Added|
|Tag Converged.Rounded With Media - RTL.default.chromium.png|0| Added|
|Tag Converged.Rounded With Media.default.chromium.png|0| Added|
|Tag Converged.Rounded With Secondary Text.default.chromium.png|0| Added|
|Tag Converged.Rounded.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
noting issues:

<img width="1491" alt="image" src="https://github.com/microsoft/fluentui/assets/28751745/8dbb3e07-a37b-4bbe-9c36-13c20ec2ef45">

Reply:
Verified